multiversx/mx-api-service
# warm scr count for smart contracts

## Reasoning
- The current implementation of fetching `scrCount` for smart contracts in `getAccountsRaw` method directly computes  the value each time, which is not efficient. This approach leads to unnecessary processing and increased response times.
  
## Proposed Changes
- Implement a caching layer for `scrCount` values of `smart contracts`. This will involve checking the cache first before computing `scrCount`, and caching the value if it's not already stored.
- Modify the `getAccountsRaw` method to fetch `scrCount` from the cache before falling back to computation. This ensures efficient data retrieval and minimizes the performance impact of fetching `scrCount` for multiple accounts.

## How to test
Pre-required steps:
- in cache warmer service, set Cron to 10 seconds

- `accounts?from=0&size=25&isSmartContract=true&withOwnerAssets=true&withDeployInfo=true&withScrCount=true` -> check if all smart contracts contains scrCount property. Do multiple requests and check response time
